docs/application/dotnet/guides/nui/imageview.md

```diff
@@ -1,53 +1,117 @@
     ```
-    imageView = new ImageView(DirectoryInfo.Resource + "images/gallery-3.jpg");
+
+2.  It is assumed that image is in the resources directory. Create an instance of the `ImageView` class and pass the path to image file as an constructor argument:
+
+    ```csharp
+    ImageView imageView = new ImageView(DirectoryInfo.Resource + "your-image.jpg");
+    Window.Instance.Add(imageView);
     ```
 
-    You can also create the `Tizen.NUI.BaseComponents.ImageView` class instance separately and define the file path by setting its `ResourceUrl` property:
+    You can also create the `ImageView` class instance separately, and define the file path by setting its `ResourceUrl` property:
 
+    ```csharp
+    imageView = new ImageView();
+    imageView.ResourceUrl = DirectoryInfo.Resource + "your-image.jpg";
+    Window.Instance.Add(imageView);
     ```
+
+2.  To change the image path after the instance has been created, use the `SetImage()` method of the `ImageView` class:
+
+    ```csharp
     imageView = new ImageView();
-    imageView.ResourceUrl = DirectoryInfo.Resource + "images/gallery-3.jpg";
+    imageView.SetImage(DirectoryInfo.Resource + "your-image.jpg");
+    Window.Instance.Add(imageView);
     ```
+## Manipulate an ImageView
+
+
+**Figure: Manipulated ImageView**
 
-2.  To change the image path after the instance has been created, use the `SetImage()` method of the `Tizen.NUI.BaseComponents.ImageView` class:
+ ![Manipulated Image View](media/imageViewManipulated.png)
 
+1.  Create object, set `Size2D` property, and display it at the center of the window by setting `ParentOrigin` and `Position2D` properties:
+
+    ```csharp
+    ImageView imageView = new ImageView(DirectoryInfo.Resource + "your-image.jpg");
+    imageView.Size2D = new Size2D(400, 400);
+    imageView.ParentOrigin = ParentOrigin.Center;
+    imageView.Position2D = new Position2D(-200, -200);
+    Window.Instance.Add(imageView);
     ```
-    imageView.SetImage(DirectoryInfo.Resource + "images/gallery-4.jpg");
+
+2.  Use `PixelArea` property to apply the zoom effect:
+
+    ```csharp
+    imageView.PixelArea = new RelativeVector4(0.25f, 0.25f, 0.5f, 0.5f);
     ```
 
-<a name="properties"></a>
-## Image View Properties
+3.  To rotate the image by 45 degrees, set the `Orientation` property as follows:
+
+    ```csharp
+    imageView.Orientation = new Rotation(new Radian((float)System.Math.PI / 4), 
+                                         new Vector3(0.0f, 0.0f, 1.0f));
+    ```
+
+## ImageView Event
+
+After all the required resources are loaded and ready, you can perform some actions:
+
+1. Create handler:
+
+    ```csharp
+    public void onResourceReady(object sender, ImageView.ResourceReadyEventArgs e)
+    {
+        // Resource is ready, do something
+    } 
+    ```
+
+2. Add handler to `ImageView` by using the `ResourceReady` property:
+
+    ```csharp
+    imageView.ResourceReady += onResourceReady;
+    ```
+
+## ImageView Properties
```

Comment:
## ImageView properties

Reply:
Hi, I don't know what's up but there should be only one commit in this PR with changes to view.md, I didn't make any changes to other files, I'm confused what has happened.